haec-user-service/test/user.e2e-spec.ts

```diff
@@ -30,4 +30,39 @@ describe('AppController (e2e)', () => {
         name: 'Mocky McMockerson',
       });
   });
+
+  it('can get a user', () => {
+    return request(app.getHttpServer())
+    .get('/user/1')
+    .expect(200)
+      .expect({
+        id: 1,
+        email: 'fake.email.com',
+        name: 'Mocky McMockerson',
+      });
+    });
+
+    it('can verify that a user passes', () => {
+      return request(app.getHttpServer())
+      .post('/user/verify')
+      .send({
+        email: 'fake.email.com',
+        password: 'password',
+      })
+      .expect(201)
+      // .expect(true)
```

Comment:
Two things here:

1. I'd get rid of the commented out `// .expect(true)`. Also, this endpoint does return data, so I'd check that too.